src/main/java/taxiservice/dao/jbdc/ManufacturerJdbcDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+package taxiservice.dao.jbdc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxiservice.dao.ManufacturerDao;
+import taxiservice.exception.DataProcessingException;
+import taxiservice.lib.Dao;
+import taxiservice.models.Manufacturer;
+import taxiservice.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer element) {
+        String query = "INSERT INTO manufacturers "
+                + "(manufacturer_name, manufacturer_country) VALUE (?, ?)";
```

Comment:
```suggestion
                + "(name, country) VALUE (?, ?)";
```
you already know that this is manufacturer